src/js/webrequest.js

```diff
@@ -1038,6 +1038,7 @@ function dispatcher(request, sender, sendResponse) {
       noTabData: false,
       origins,
       seenComic: badger.getSettings().getItem("seenComic"),
+      showExpandedTrackingSection: badger.getSettings().getItem("showExpandedTrackingSection"),
```

Comment:
Link tracking protections section is in a strange state where it takes two clicks to expand it when `showExpandedTrackingSection` is `true`.

Reply:
I'm unable to recreate this bug :/
Are you still able to produce it?